mod.conf

```diff
@@ -1,3 +1,4 @@
 name = throwing
+depends = farming
```

Comment:
Put it as an optional dependency as it's an API mod and works just fine without farming.

Reply:
It should at least be mentioned in `README.md` that enabling farming mod is required to have bows crafts.
If we do this, maybe 
```lua
	if def.itemcraft then
		-- Check for an override name
		-- because an ``output = ":name"'' can't exist in a recipe
		local output_name = name
		if name:sub(1,1) == ":" then
			output_name = name:sub(2)
		end
		minetest.register_craft({
			output = output_name,
			recipe = {
				{"farming:cotton", def.itemcraft, ""},
				{"farming:cotton", "", def.itemcraft},
				{"farming:cotton", def.itemcraft, ""},
			}
		})
	end
```
should be changed by

```lua
	if def.itemcraft and minetest.get_modpath("farming") then
		-- Check for an override name
		-- because an ``output = ":name"'' can't exist in a recipe
		local output_name = name
		if name:sub(1,1) == ":" then
			output_name = name:sub(2)
		end
		minetest.register_craft({
			output = output_name,
		recipe = {
				{"farming:cotton", def.itemcraft, ""},
				{"farming:cotton", "", def.itemcraft},
				{"farming:cotton", def.itemcraft, ""},
			}
		})
	end
```

Reply:
See 4f8ab0a2a38831b8e9050fae1828f9eb2b995bd9.

Reply:
https://github.com/minetest-mods/throwing/commit/1449f49af711469ed15cf7355dfeb33e1af4825c
`throwing` no longer registers the crafts: this has been relegated to throwing, so that there is no longer an issue about which dependencies are appropriate for which mods.